23.md

```diff
@@ -0,0 +1,91 @@
+NIP-23
+======
+
+Relays List
+-----------
+
+`draft` `optional` `author:fiatjaf` `author:cameri` `author:monlovesango` `author:giszmo`
+
+A special event with kind `10001`, meaning "relay list" is defined as having a list of tags, one for each relay the author uses.
+
+The content is not used.
+
+The tags consist of arrays of 3 elements: the first is the relay URL, the second is the _read_ condition, the third is the _write_ condition.
+
+The _read_ condition consists of a string containing a rule that follows a subset of the [runes](https://pypi.org/project/runes/) language. Specifically only the `|`, `&`, `!`, `>`, `<`, `/` and `=` operators are allowed. When the rule is an empty string it evaluates to `true`. All the operators must be tested against all possible values in the case of filter values that consist of lists of values and also in the case of event tags that can have multiple values for the same tag -- in other words, the `=` may be interpreted as a `values.any(v => v == runeValue)` instead of an `values == runeValue` operator; the `<` may be interpreted as a `values.any(v => v < runeValue)` and so on. Event tags are identified just by their tag key (for example, `e` or `p`) without the `#` prefix used in filters.
```

Comment:
Why not include starts-with and ends-with from runes? Would make rules more compact with hardly any false positives and using libraries, implementers wouldn't care.

Reply:
I agree, please add starts with. For Ends with I can't think of a use case right now.

Reply:
Sure.

Reply:
@Cameri `endsWith` because proof of work in event IDs makes the start less attractive to use as a short identifier.

Reply:
should we mention here that the underscore `_` is an allowed identifier? It's needed for `created_at`

Reply:
@fiatjaf bumping this